Makefile

```diff
@@ -147,3 +147,18 @@ clean:
 clean-apigenv3:
 	@echo "removing artifacts from openapi v3 generator"
 	@cd api/gen/v1alpha && rm -rf .swagger-codegen/ && rm -f README.md && rm -f .swagger-codegen-ignore
+
+.PHONY: lint
+lint:
+	$(DOCKER) run -t --rm -v $(PWD):/app -w /app golangci/golangci-lint golangci-lint run -v
```

Comment:
1) I often use `golangci-lint run --fix` locally. Could you also add a target for that one? `make lint-fix` or sth?
2) may be good to pin the version to the version? wdyt?

Reply:
1. Would lint-fix be better integrated into our IDEs though? I'm open to being persuaded on this, though.
2. I didn't pin the version because, and defaulted to latest, because that's what we do in our github action right now: https://github.com/RedHatInsights/authz/blob/506d3f28f37cc79c451a12dd16be107938bdcc59/.github/workflows/golangci-lint.yml#L22.

Reply:
Re versions, I suppose I'm guessing that the golangci-lint version in the latest golangci-lint docker image will be the same as the golangci-lint version picked up by https://github.com/golangci/golangci-lint-action, more or less. If so, I wouldn't expect too many issues, and at least we don't have to maintain a linter version.